src/Controller/Api/CategoryController.php

```diff
@@ -256,7 +256,7 @@ protected function handleWriteCategory($request, $id = null)
 
             $view = FOSRestView::create($category);
 
-            if (class_exists('FOS\RestBundle\Context\Context')) {
+            if (class_exists(Context::class)) {
```

Comment:
Import is missing

Reply:
It was already there https://github.com/sonata-project/SonataClassificationBundle/blob/3.x/src/Controller/Api/CategoryController.php#L14